### DISCUSSION TITLE
Rotor-Flying Manipulator Simulation (Project 47)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#12] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #47](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Rotor-Flying%20Manipulator%20Simulation)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by yatinsd** [comment]
Hi, 
I am very much interested in doing this project. I am from Cranfield University, and this is my first time doing collaborative robotics and am open to collaborations, if anyone is interested kindly let me know?

**Reply by mohit-cloud** [reply]
hey, I am an undergraduate student from NIT Jamshedpur, India, and if you are still interested, I'd be very much interested to work together in this...

**Comment by yatinsd** [comment]
Hi Mohit,
That's great news! Which engineering undergraduate you're doing now?

**Reply by mohit-cloud** [reply]
I study mechanical engineering there

**Comment by yatinsd** [comment]
Cab we take this over personal message please?

**Reply by mohit-cloud** [reply]
??

**Comment by robertogl** [comment]
Hi All,

If you are planning to use Simscape to model your system, you may want to take a look at this repository!
https://github.com/mathworks/Quadcopter-Drone-Model-Simscape

Enjoy!
